content/source/docs/cloud/sentinel/manage-policies.html.md

```diff
@@ -64,18 +64,74 @@ Terraform Cloud has support for Sentinel's modules feature. This allows you to w
+
+Create a `modules` directory within the policy set working directory, and create a `timezone.sentinel` that looks as follows:
+
+```sentinel
+import "http"
+import "json"
+import "strings"
+
+uri = "https://raw.githubusercontent.com/dmfilipenko/timezones.json/master/timezones.json"
+request = http.get(uri)
+response = json.unmarshal(request.body)
+
+offset = func(abbr) {
+	timezone = filter response as _, r {
+		r.abbr is strings.to_upper(abbr)
+	}
+	for timezone as tz {
+		print("Getting timezone data for", tz.value)
+		print("Details:", tz.text)
+		print("Abbreviation:", tz.abbr)
+		print("Offset:", tz.offset)
+		return tz.offset
+	}
+}
+```
 
--> **NOTE:** At this point in time, you cannot load modules from outside of policy set working directory hierarchy. This means in the above example, a `source` of `../modules/foo.sentinel` will not work.
+The above configuration would tell a policy check to load the code at `./modules/timezone.sentinel` relative to the policy set working directory and make it available to be imported with the statement `import "timezone"`, located at the top of the Sentinel policy code. This module will be available to all of the policies within the policy set.
 
 ### Sentinel policy code files
 
-Sentinel policies themselves are defined in individual files (one per policy) in the same directory as the `sentinel.hcl` file. These files must match the name of the policy from the configuration file and carry the `.sentinel` suffix. Using the configuration example above, a policy file named `sunny-day.sentinel` should also exist alongside the `sentinel.hcl` file to complete the policy set.
+Sentinel policies themselves are defined in individual files (one per policy) in the same directory as the `sentinel.hcl` file. These files must match the name of the policy from the configuration file and carry the `.sentinel` suffix. Using the configuration example above, a policy file named `enforce-terraform-maintenance-windows.sentinel` should also exist alongside the `sentinel.hcl` file to complete the policy set. 
+
+Using the `enforce-terraform-maintenance-windows.sentinel` policy as an example, we can use the `time` and `tfrun` imports along with our custom `timezone` module to enforce checks that:
+
+1. Load the time when the Terraform run occurred
+1. Convert the loaded time to PST
+1. Verify that the provisioning operation is only going to occur on an agreed upon day
+
+An example policy would be as follows:
+
+```sentinel
+import "time"
+import "tfrun"
+import "timezone"
+
+param maintenance_days default ["Friday", "Saturday", "Sunday"]
+param timezone_abbreviation default "PST"
+
+tfrun_created_at = time.load(tfrun.created_at)
+
+supported_maintenance_day = rule {
+	tfrun_created_at.add(time.hour * timezone.offset(timezone_abbreviation)).weekday_name in maintenance_days
```

Comment:
My only beef with this example is that it suffers the same temporal issues we have known about for a long time. If you plan this run on Saturday, the policy will pass, but nothing stops you from applying it on Wednesday. Personally I'd like to see us back away from this style of example until that's addressed. The description of the policy is actually inaccurate because of this as well.

Reply:
Note that this is particularly true because this is specifically the Terraform Cloud documentation, where policies are currently only evaluated at plan time and not on an apply.

Reply:
@ryanuber based on your feedback I've updated the policy to use a rule expression and the `when` predicate so the rule will evaluate on a workspace that has `auto_apply` enabled.

```
supported_maintenance_day = rule when tfrun.workspace.auto_apply is true {
	tfrun_created_at.add(time.hour * timezone.offset(timezone_abbreviation)).weekday_name in maintenance_days
}
```

I do see value in this approach and I _think_ it addresses your concern.